### PR TITLE
Moved agenda tree code to the Item-Manager and used it at the AgendaPDFView.

### LINKED ISSUE #1561: AttributeError at /agenda/print/
Create some new agenda items and generate agenda PDF.

=> AttributeError at /agenda/print/
'Item' object has no attribute 'get_ancestors'

### LINKED ISSUE #1571: Autoupdate on agenda tree
At the moment the agenda tree is not updated when the weight or the parent of one item changes. We have to find a way to update data from non-model-instances.

### PR DESCRIPTION
**Pull request by ostcar**
fixed #1561

### CODE REVIEW

File: openslides/agenda/views.py [hunk 1]
```diff
@@ -218,64 +227,10 @@ def tree(self, request):
...
-        else:
-            # Everithing is fine. Return a response with status_code 200 an no content
-            return Response()
-        return Response({'detail': detail}, status=400)
+            try:
+                Item.objects.set_tree(request.data['tree'])
+            except ValueError as error:
+                return Response({'detail': str(error)}, status=400)
+            else:
+                return Response()
```
Comment (normanjaeckel) [inline openslides/agenda/views.py]: Do you want to add a short success message here?

`Response({'detail': 'Agenda tree successfully updated.'})`

File: openslides/agenda/models.py [hunk 2]
```diff
@@ -17,10 +18,75 @@
 from openslides.utils.utils import to_roman
 
 
+class ItemManager(models.Manager):
+    def get_tree(self, only_agenda_items=False, include_content=False):
+        """
+        Generator that yields dictonaries. Each dictonarie has two keys, id
```
Comment (normanjaeckel) [inline openslides/agenda/models.py]: Typo `Each dictonary`

File: openslides/agenda/models.py [hunk 3]
```diff
@@ -17,10 +18,75 @@
...
+        and children, where id is the id of one agenda item and children is a
+        generator that yields dictonaries like the one discribed.
+
+        If only_agenda_items is True, the tree hides ORGANIZATIONAL_ITEMs.
+
+        If include_content is True, the yielded dictonaries have no key 'id'
+        but a key 'item' which the entire item object.
```
Comment (normanjaeckel) [inline openslides/agenda/models.py]: Typo `with the entire object.` or `which is the ...`

**Comment by normanjaeckel** [comment]
Typos and proposal. Then ready to merge.

File: openslides/agenda/views.py (hunk 1 above)
Comment (ostcar) [inline openslides/agenda/views.py]: done

File: openslides/agenda/models.py (hunk 2 above)
Comment (ostcar) [inline openslides/agenda/models.py]: done

File: openslides/agenda/models.py (hunk 3 above)
Comment (ostcar) [inline openslides/agenda/models.py]: done

**Pull request merged**